test/controllers/admin/bill_runs.controller.test.js

```diff
@@ -0,0 +1,91 @@
+'use strict'
+
+// Test framework dependencies
+const Lab = require('@hapi/lab')
+const Code = require('@hapi/code')
+const Sinon = require('sinon')
+
+const { describe, it, before, beforeEach, after, afterEach } = exports.lab = Lab.script()
+const { expect } = Code
+
+// For running our service
+const { deployment } = require('../../../server')
+
+// Test helpers
+const {
+  AuthorisationHelper,
+  AuthorisedSystemHelper,
+  BillRunHelper,
+  DatabaseHelper,
+  RegimeHelper
+} = require('../../support/helpers')
+
+// Things we need to stub
+const JsonWebToken = require('jsonwebtoken')
+const { SendTransactionFileService } = require('../../../app/services')
+
+describe('Presroc Bill Runs controller', () => {
+  const clientID = '1234546789'
+  let server
+  let authToken
+  let regime
+  let authorisedSystem
+  let billRun
+  let sendStub
+
+  before(async () => {
+    server = await deployment()
+    authToken = AuthorisationHelper.nonAdminToken(clientID)
+
+    Sinon
+      .stub(JsonWebToken, 'verify')
+      .returns(AuthorisationHelper.decodeToken(authToken))
+  })
+
+  beforeEach(async () => {
+    await DatabaseHelper.clean()
+
+    regime = await RegimeHelper.addRegime('wrls', 'WRLS')
+    authorisedSystem = await AuthorisedSystemHelper.addSystem(clientID, 'system1', [regime])
+
+    sendStub = Sinon.stub(SendTransactionFileService, 'go')
+  })
+
+  after(async () => {
+    Sinon.restore()
+  })
+
+  afterEach(async () => {
+    sendStub.restore()
+  })
+
+  describe('Send bill run: PATCH /admin/{regimeId}/bill-runs/{billRunId}/send', () => {
+    const options = (token, billRunId) => {
+      return {
+        method: 'PATCH',
+        url: `/admin/wrls/bill-runs/${billRunId}/send`,
+        headers: { authorization: `Bearer ${token}` }
+      }
+    }
+
+    beforeEach(async () => {
+      billRun = await BillRunHelper.addBillRun(authorisedSystem.id, regime.id)
```

Comment:
The only tweak I'd make in this PR is here. To better represent the scenario we expect to cover I'd use this opportunity to set the status of the test bill run to pending.

```suggestion
      billRun = await BillRunHelper.addBillRun(authorisedSystem.id, regime.id, 'A', 'pending')
```